Libplanet.Net.Tests/TestUtils.cs

```diff
@@ -236,6 +237,14 @@ in PrivateKeys.Zip(Peers, (first, second) => (first, second)))
             policy ??= Policy;
             var fx = new MemoryStoreFixture(policy.BlockAction);
             var blockChain = CreateDummyBlockChain(fx);
+            for (int i = 1; i < height; i++)
+            {
+                var block = blockChain.ProposeBlock(
+                    Libplanet.Tests.TestUtils.GenesisProposer,
+                    lastCommit: CreateBlockCommit(blockChain[i - 1]));
+                blockChain.Append(block, CreateBlockCommit(block));
+            }
+
```

Comment:
Could you change the `PrivateKey` field so that the blocks can have the proposer in a round-robin order, for now? 
Or, We could fix it later because we will not have a reward until the PoS, so instead flag it here as TODO.

Reply:
What I've thought on that time was, unit test doesn't need to be work as real work, but now my mind has been changed.
May be this kind of mock up can be harmful especially after future sortition logic implementation.
I think it would be better to make some fixture like `ValidatorSet` that have private keys also, for tests.
I'll add some.

Reply:
Instead of create new fixture, added dictionary from `publicKey` to `privateKey` on `TestUtils`. And, proposer is now selected with `ValidatorSet.GetProposer()`.